src/Symfony/Component/Security/Guard/Token/PreAuthenticationGuardToken.php

```diff
@@ -58,7 +58,7 @@ public function getCredentials()
         return $this->credentials;
     }
 
-    public function setAuthenticated($authenticated)
+    public function setAuthenticated(bool $authenticated)
```

Comment:
💡 This change to `security-guard` 5 breaks compatibility with `security-core` 4.4, thus the bump in composer.json. Can be reverted and re-applied later, if you don't want to bump yet.